doc/sphinxext/cited_mne.py

```diff
@@ -37,12 +44,12 @@
       'Gecko/20100913 Firefox/3.6.9')
 
 # ##### Templates for citations #####
-html = (u""".. _cited
+html = (u""".. _cited:
 
 Publications by users
 =====================
 
-Papers citing MNE as extracted from Google Scholar (on %s).
+Papers citing MNE (%d) as extracted from Google Scholar (on %s).
```

Comment:
now I realized that maybe the numbering should have been left out from `cited.rst`. That way, you'll avoid generating a big diff every time it's updated ...